servlet/src/main/java/com/linecorp/armeria/server/servlet/DefaultServletHttpResponse.java

```diff
@@ -0,0 +1,339 @@
+    public String encodeURL(String url) {
+        requireNonNull(url, "url");
+        return url;
+    }
+
+    @Override
+    public String encodeRedirectURL(String url) {
+        requireNonNull(url, "url");
+        return encodeURL(url);
+    }
+
+    @Override
+    @Deprecated
+    public String encodeUrl(String url) {
+        requireNonNull(url, "url");
+        return encodeURL(url);
+    }
+
+    @Override
+    @Deprecated
+    public String encodeRedirectUrl(String url) {
+        requireNonNull(url, "url");
+        return encodeRedirectURL(url);
+    }
+
+    @Override
+    public void sendError(int sc, @Nullable String msg) throws IOException {
+        if (msg == null) {
+            msg = "";
+        }
+        headersBuilder.status(new HttpStatus(sc, msg));
+        if (responseWriter.tryWrite(headersBuilder.build())) {
+            if (responseWriter.tryWrite(HttpData.ofUtf8(msg))) {
+                responseWriter.close();
+            }
+        }
+    }
+
+    @Override
+    public void sendError(int sc) throws IOException {
+        sendError(sc, null);
+    }
+
+    @Override
+    public void sendRedirect(String location) throws IOException {
+        requireNonNull(location, "location");
+        if (responseWriter.tryWrite(
+                ResponseHeaders.of(HttpStatus.SEE_OTHER, HttpHeaderNames.LOCATION.toString(), location))) {
+            responseWriter.close();
+        }
+    }
+
+    @Override
+    public void setDateHeader(String name, long date) {
+        requireNonNull(name, "name");
+        checkArgument(date > 0, "date: %s (expected: > 0)", date);
+        headersBuilder.setObject(name, date);
+    }
+
+    @Override
+    public void addDateHeader(String name, long date) {
+        requireNonNull(name, "name");
+        checkArgument(date > 0, "date: %s (expected: > 0)", date);
+        headersBuilder.addObject(name, date);
+    }
+
+    @Override
+    public void setHeader(String name, String value) {
+        requireNonNull(name, "name");
+        requireNonNull(value, "value");
+        headersBuilder.setObject(name, value);
+    }
+
+    @Override
+    public void addHeader(String name, String value) {
+        requireNonNull(name, "name");
+        requireNonNull(value, "value");
+        headersBuilder.addObject(name, value);
+    }
+
+    @Override
+    public void setIntHeader(String name, int value) {
+        requireNonNull(name, "name");
+        checkArgument(value >= 0, "value: %s (expected: >= 0)", value);
+        headersBuilder.setInt(name, value);
+    }
+
+    @Override
+    public void addIntHeader(String name, int value) {
+        requireNonNull(name, "name");
+        checkArgument(value >= 0, "value: %s (expected: >= 0)", value);
+        headersBuilder.addInt(name, value);
+    }
+
+    @Override
+    public void setContentType(String contentType) {
+        requireNonNull(contentType, "contentType");
+        setHeader(HttpHeaderNames.CONTENT_TYPE.toString(), contentType);
```

Comment:
Could just call directly `headersBuilder.set(HttpHeaderNames.CONTENT_TYPE, contentType);`
So we don't have to call `toString`